docs/modules/ROOT/pages/vshn-managed/postgresql/restore.adoc

```diff
@@ -0,0 +1,53 @@
+= Restore a PostgreSQL service
+
+IMPORTANT: The restores can only be triggered from an instance in the same namespace.
+If the old instance has already been deleted the backups are deleted as well.
+
+. Get the name of the backup you want to restore by listing all available backups.
++
+[source,bash]
+----
+kubectl -n postgres get vshnpostgresbackups
+BACKUP NAME                                 DATABASE INSTANCE   STORED TIME               STATUS     AGE
+
+pgsql-app1-prod-pk8k4-2023-03-05-13-05-00   pgsql-app1-prod     2023-03-05T13:05:15.755Z  Completed  38s
+pgsql-app1-dev-pk8k4-2023-03-01-16-52-11    pgsql-app1-dev                                Failed     28s
+----
+NOTE: The list will show all backups from all instances in this namespace. In the example above there are two database instances with one backup each.
+
+
+. Apply the following object on your namespace, as specified by its YAML description.
++
+.Example to restore a PostgreSQL instance
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNPostgreSQL
+metadata:
+  name: pgsql-app1-restore
+  namespace: postgres
+spec:
+  parameters:
+    restore:
+      claimName: pgsql-app1-prod <1>
```

Comment:
Is there a reason why we - again - use the term "claim" here? I thought we had a discussion earlier that this term is specific to Crossplane and not for the end-user, as it's not used elsewhere and we also don't use it in other user-facing APIs afaik.

I guess this is namespaced? I mean, could it be a security issue and being able to take over other already existing instances?

Reply:
> Is there a reason why we - again - use the term "claim" here? I thought we had a discussion earlier that this term is specific to Crossplane and not for the end-user, as it's not used elsewhere and we also don't use it in other user-facing APIs afaik.

I agree with you, we should not use crossplane terms where possible. Unfortunately we do haven't updated the claim yet so I will have to open a task for that. Meanwhile is it ok to leave it as is?

> I guess this is namespaced? I mean, could it be a security issue and being able to take over other already existing instances?

This is namespace scoped therefore the instances are accessed by any user having access to the namespace. Personally I don't see a problem here. If the customer wants different access to his databases then he should create them in different namespaces.